benchopt/plotting/html/static/benchmark.js

```diff
@@ -0,0 +1,175 @@
+$(document).ready(function () {
+  // Sort the table by descending date order (date being in column 0)
+  $(".summary").dataTable({
+    order: [[0, "desc"]],
+  }); // reorder table by date
+  if (location.hostname === "") {
+    // local file and not doc website
+    $("[name='checkfiles']").css({
+      // only show lines that not hidden by user
+      display: "block",
+      "margin-left": "auto",
+      float: "left",
+    }); // display checkboxes
+    $("#trashBtn").css({
+      display: "block",
+      float: "left",
+      "margin-bottom": "5vh",
+    });
+  }
+});
+
+// hide columns without content (if system informations not available)
+$("table").each(function (a, tbl) {
+  var currentTableRows = $(tbl).find("tbody tr").length;
+  $(tbl)
+    .find("th") // also take care of table head
+    .each(function (i) {
+      var remove = 0;
+      var currentTable = $(this).parents("table");
+      var tds = currentTable.find("tr td:nth-child(" + (i + 1) + ")");
+      tds.each(function (j) {
+        if ($(this)[0].innerText.length == 0) remove++;
+      });
+      if (remove == currentTableRows) {
+        $(this).hide();
+        tds.hide();
+      }
+    });
+});
+
+/**
+ * Filter table from dropdown current value
+ */
+function change(ll_item) {
+  var td, i, fil;
+  var filter = new Array();
+  for (item = 0; item < ll_item.length; item++) {
+    filter.push(
+      document
+        .getElementById("select" + ll_item[item].replace(/\s/g, ""))
+        .value.toUpperCase()
+    ); // Add dropdown value on top of the array (order is necessary)
+  }
+  var table = document.getElementById("summary");
+  var tr = table.getElementsByTagName("tr");
+  for (i = 0; i < tr.length; i++) {
+    tr[i].style.display = "";
+    td = tr[i].getElementsByTagName("td")[2];
+    if (td) {
+      // do not display elements from table not in filter
+      for (fil = 0; fil < filter.length; fil++) {
+        if (
+          td.innerHTML.toUpperCase().indexOf(filter[fil]) > -1 &&
+          tr[i].style.display !== "none"
+        ) {
+          tr[i].style.display = "";
+        } else {
+          tr[i].style.display = "none";
+        }
+      }
+    }
+  }
+}
+
+/**
+ * Display subinfo in the table
+ * Toggle between + and - icon button
+ */
+function displaymore(id, loop_index) {
```

Comment:
For consistent naming:
```suggestion
function displayMore(id, loop_index) {
```

Reply:
In [tfae2072 commit](https://github.com/benchopt/benchopt/pull/232/commits/fae20727268085a2cac52e5ef78fa99bbe1bd33d)